From: gaosibei <gaosb@inspur.com>
Date: Tue, 26 Sep 2017 10:19:22 +0800
Subject: RGW: fix a bug about inconsistent unit of comparison
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Fixesï¼šhttp://tracker.ceph.com/issues/21590
Resolves (with following): rhbz#1498280

Signed-off-by: gaosibei <gaosb@inspur.com>
(cherry picked from commit cb39f065d6eba87cf1fc2e99334322bf63092df3)
Signed-off-by: Matt Benjamin <mbenjamin@redhat.com>
(cherry picked from commit 2ba24bfad7086661ca1050114dc1bdbb544f5013)
---
 src/rgw/rgw_quota.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/rgw/rgw_quota.cc b/src/rgw/rgw_quota.cc
index dc880b4..9ef1eba 100644
--- a/src/rgw/rgw_quota.cc
+++ b/src/rgw/rgw_quota.cc
@@ -106,9 +106,9 @@ bool RGWQuotaCache<T>::can_use_cached_stats(RGWQuotaInfo& quota, RGWStorageStats
       quota.max_size_soft_threshold = quota.max_size_kb * store->ctx()->_conf->rgw_bucket_quota_soft_threshold;
     }
 
-    if (cached_stats.num_kb_rounded >= (uint64_t)quota.max_size_soft_threshold) {
+    if (cached_stats.size_rounded  >= (uint64_t)quota.max_size_soft_threshold) {
       ldout(store->ctx(), 20) << "quota: can't use cached stats, exceeded soft threshold (size): "
-        << cached_stats.num_kb_rounded << " >= " << quota.max_size_soft_threshold << dendl;
+        << cached_stats.size_rounded << " >= " << quota.max_size_soft_threshold << dendl;
       return false;
     }
   }
